src/ui/server/helpers/summary-lists/company-house-summary-list.ts

```diff
@@ -6,21 +6,35 @@ import fieldGroupItem from './generate-field-group-item';
 import getFieldById from '../get-field-by-id';
 import { ApplicationExporterCompany, CompanyHouseResponse, SummaryListItemData } from '../../../types';
 import generateMultipleFieldHtml from '../generate-multiple-field-html';
+import { stringArrayHasValue, isPopulatedArray } from '../array';
 
 const {
   EXPORTER_BUSINESS: { COMPANY_HOUSE },
 } = FIELD_IDS.INSURANCE;
 
-const { COMPANY_NAME, COMPANY_ADDRESS, COMPANY_NUMBER, COMPANY_INCORPORATED, COMPANY_SIC } = COMPANY_HOUSE;
+const { COMPANY_NAME, COMPANY_ADDRESS, COMPANY_NUMBER, COMPANY_INCORPORATED, COMPANY_SIC, COMPANY_SIC_DESCRIPTION } = COMPANY_HOUSE;
 
 /**
  * generateSicCodesValue
- * @param {String} Sic codes
+ * generates string with sicCode and description with line breaks
+ * maps through sicCodes, and if there is a description at that index, string will contain description
+ * @param {Array<string>} sicCodes
+ * @param {Array<string>} sicCodeDescriptions
  * @returns {String} Sic codes as a single string or default empty string
  */
-const generateSicCodesValue = (sicCodes?: Array<string>): string => {
-  if (sicCodes && sicCodes.length) {
-    return sicCodes.toString();
+const generateSicCodesValue = (sicCodes?: Array<string>, sicCodeDescriptions?: Array<string>): string => {
+  if (sicCodes && isPopulatedArray(sicCodes)) {
```

Comment:
Can just be `if (isPopulatedArray(sicCodes)) { ...`

Do we need the same for `sicCodeDescriptions`?

Reply:
lint does not like it, so needed to put the `sicCodes &&` in front, linting otherwise says could be null or undefined on the lines below

Reply:
Would this work?


```js
if (!isPopulatedArray(sicCodes)) {
  return DEFAULT.EMPTY;
}
```

Reply:
linting still complains at the `sicCodes.forEach((sicCode, index) => {`